packages/ui-components/eslint.config.mjs

```diff
@@ -4,15 +4,19 @@
  */
 
 import junoConfigs from "@cloudoperators/juno-config/eslint/juno-typescript.mjs"
+import { RuleTester } from "eslint"
 
 export default [
   ...junoConfigs,
   {
     files: ["test/__mocks__/svgLib.js", "src/**/*.test.js"],
     languageOptions: { sourceType: "module" },
   },
-
   {
-    ignores: ["setupTests.js"],
+    rules: {
```

Comment:
What was the reason to apply it on ui-components lvl?

Reply:
We should consider adding those rules to the base configuration in the imported config package